src/tools/rbd/Utils.cc

```diff
@@ -832,10 +832,26 @@ int get_formatter(const po::variables_map &vm,
                 << "is json or xml" << std::endl;
       return -EINVAL;
     }
+  } else if (vm[at::PRETTY_FORMAT].as<bool>()) {
+    std::cerr << "rbd: --pretty-format only works when --format "
+              << "is json or xml" << std::endl;
+    return -EINVAL;
   }
   return 0;
 }
 
+int add_newline_at_end(const po::variables_map &vm,
```

Comment:
I was suggesting adding a boolean setter to the ```Formatter``` that would be enabled by the rbd CLI if using json / xml non-pretty formatter.